ktxspec.adoc

```diff
@@ -1071,6 +1071,21 @@ have shaders that understand this channel layout.
 The bitstream of the UASTC data is described in Chapter 25 _UASTC
 Compressed Texture Image Format_ of <<KDF13>>.
 
+Basis Universal UASTC HDR Format::
+The Universal ASTC HDR image format (UASTC HDR) is indicated by
+`colorModel` `KHR_DF_MODEL_UASTC_HDR` (= 167) together with `vkFormat`
+`VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK` (= 1000066000). The DFD must be
+as described in Section ??? of <<KDF13>>. Images in this format are
+a strict RGB-only ASTC HDR subset that must be transcoded to BC6H
+or decoded to raw 16-bit floating-point values before being uploaded
+to and sampled by a GPU that does not support ASTC. UASTC HDR
```

Comment:
I think it's important to explicitly mention "does not support ASTC HDR" here because some vendors still ship GPUs that support only ASTC LDR.

Reply:
Excellent point. Thanks.

Reply:
Resolved in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/2e3fec435d62d39e286405d28d0efb3725ed8eb6.